tsdb/index/index.go

```diff
@@ -1076,24 +1122,43 @@ func (s Symbols) ReverseLookup(sym string) (uint32, error) {
 	return uint32(s.bs.Len() - lastLen), nil
 }
 
-func (s Symbols) All() (map[string]struct{}, error) {
+func (s Symbols) Size() int {
+	return len(s.index) * 8
+}
+
+func (s Symbols) Iter() StringIter {
```

Comment:
Worth to mention the yoloString consequence? (string is valid until the byteSlice data is still mmapped)?

EDIT: Already done on `Symbols` method

Reply:
Did I miss any on the main DB APIs?

Reply:
All good, see EDIT